pyproject.toml

```
@@ -9,22 +9,23 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "ndcube"
 description = "A package for multi-dimensional contiguous and non-contiguous coordinate aware arrays."
-requires-python = ">=3.10"
+requires-python = ">=3.12"
 readme = { file = "README.rst", content-type = "text/x-rst" }
 license = { file = "licenses/LICENSE.rst" }
 authors = [
   { name = "The SunPy Community", email = "sunpy@googlegroups.com" },
 ]
 dependencies = [
-  "astropy>=5.0.6,!=5.1.0",
-  "gwcs>=0.18",
-  "numpy>=1.23.0",
-  "scipy>=1.8.0",
+  "astropy>=6.1.0",
```

We're kinda on the line for 2 years, 6.0 is Nov 27th.
```suggestion
  "astropy>=5.3.0",
```